crates/astria-proto/src/native/sequencer/mod.rs

```diff
@@ -1 +1,2 @@
+pub mod asset;
```

Comment:
Do you expect that these need not be versioned?

Reply:
not necessarily, i just really don't like the 1000+ line v1alpha1.rs file lol. can we turn that into a module and put the types in separate files?

Reply:
Haha, yeah. I think it'd be fine to split up `v1alpha` into modules.